src/components/hmi_message_handler/src/websocket_session.cc

```diff
@@ -102,7 +104,7 @@ void WebsocketSession::Send(std::string& message, Json::Value& json_message) {
 }
 
 void WebsocketSession::sendJsonMessage(Json::Value& message) {
-  std::string str_msg = m_writer.write(message);
+  std::string str_msg = Json::writeString(m_writer, message);
```

Comment:
@ydementieiev make it const if you don't change it afterwards

Reply:
@AByzhynar done in [1345b2e](https://github.com/smartdevicelink/sdl_core/pull/3155/commits/1345b2e4f52415e896a5fa3d232aa391c69751bb)